x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/crawler_single_domain.tsx

```diff
@@ -32,23 +32,21 @@ export const CrawlerSingleDomain: React.FC = () => {
 
   const { fetchDomainData } = useActions(CrawlerSingleDomainLogic);
 
-  const displayDomainUrl = domain
-    ? domain.url
-    : i18n.translate('xpack.enterpriseSearch.appSearch.crawler.singleDomain.loadingTitle', {
-        defaultMessage: 'Loading...',
-      });
-
   useEffect(() => {
     fetchDomainData(domainId);
   }, []);
 
   return (
     <AppSearchPageTemplate
-      pageChrome={getEngineBreadcrumbs([CRAWLER_TITLE, displayDomainUrl])}
-      pageHeader={{
-        pageTitle: displayDomainUrl,
-        rightSideItems: [<ManageCrawlsPopover />, <CrawlerStatusIndicator />],
-      }}
+      pageChrome={getEngineBreadcrumbs([CRAWLER_TITLE, domain?.url || '...'])}
+      pageHeader={
+        dataLoading
+          ? undefined
```

Comment:
Also a pattern used in our codebase - I think more in WS than AS, since AS has more names that can be derived from the URL vs IDs